app/controllers/posts_controller.rb

```diff
@@ -163,7 +167,9 @@ def events
             event_type: 'booking',
             appointment_type: '',
             patient_name: 'Rutuja Khanpekar',
-            subject: 'boo'
+            subject: 'boo',
+            prepay_amount: 0,
+            is_booked: false
```

Comment:
FYI , if the `prepay_amount` is `0` then the `is_paid` flag will always be `false`